source/deploying-with-kubernetes/kubernetes-upgrade.rst

```diff
@@ -0,0 +1,212 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _kubernetes_upgrade:
+
+Upgrade Wazuh installed in Kubernetes
+=====================================
+
+When upgrading our version of Wazuh installed in Kubernetes we must follow this steps.
+
+
+Check which files are exported to the volume
+--------------------------------------------
+
+Our Kubernetes deployment uses our Wazuh images from Docker. If we look at the following code extracted from the Wazuh configuration using Docker we can see which directories and files are used in the upgrade.
+
+.. code-block:: bash
+
+    DATA_DIRS[((i++))]="api/configuration"
+    DATA_DIRS[((i++))]="etc"
+    DATA_DIRS[((i++))]="logs"
+    DATA_DIRS[((i++))]="queue/db"
+    DATA_DIRS[((i++))]="queue/rootcheck"
+    DATA_DIRS[((i++))]="queue/agent-groups"
+    DATA_DIRS[((i++))]="queue/agent-info"
+    DATA_DIRS[((i++))]="queue/agents-timestamp"
+    DATA_DIRS[((i++))]="queue/agentless"
+    DATA_DIRS[((i++))]="queue/cluster"
+    DATA_DIRS[((i++))]="queue/rids"
+    DATA_DIRS[((i++))]="queue/fts"
+    DATA_DIRS[((i++))]="var/multigroups"
+
+Any file that we modify referring to the files previously mentioned, will be changed also the corresponding volume. When the corresponding Wazuh pod is created again, it will get the cited files from the volume, thus keeping the changes made previously.
```

Comment:
Rework this paragraph, it has some spell typos and doesn't sound like well versed English.